build: drop obj/ subdirectory for generated build.h

theStack opened the pull request:
As indicated by the TODO, the obj subdirectory is not needed anymore now for the generated build.h header, since autotools are gone and we don't have in-source builds anymore (see #30454, #30664). In the second commit the superflous `HAVE_BUILD_INFO` macro is dropped, as suggested in https://github.com/bitcoin/bitcoin/pull/30856#pullrequestreview-2292424496.

theStack committed:
build: drop obj/ subdir for generated build.h, rename to bitcoin-build-info.h

Now that this file is not in a subfolder anymore, prefix it with
"bitcoin-" to avoid potential collisions. Also add "info" for a more
descriptive name.

hebasto reviewed (approved):
ACK 820bae4708b013a1634bc62e0c6b56c9688ac645.

I wonder if we can get rid of the `HAVE_BUILD_INFO` macro simultaneously or in a follow-up.

theStack committed:
build: drop superfluous `HAVE_BUILD_INFO` define

bitcoin-build-info.h should always be generated before clientversion.cpp
is compiled due to the following explicit dependency in src/CMakeLists.txt:

add_dependencies(bitcoin_clientversion generate_build_info)

Hence there is no need to gate the inclusion of that header with an
extra define.

fanquake commented:
Could you rebase this, now that guix builds are reproducible again.

fanquake commented:
Guix Build
```bash
d4baa1d2e9ee7abeb60d115dd52c6bc7d02e8116fe4b94c0f040371321e37b80  guix-build-253a691441dd/output/aarch64-linux-gnu/SHA256SUMS.part
7c38410b9468799b74371ba195f34121a4789e917dcf611f2ccc4b81f8a63eee  guix-build-253a691441dd/output/aarch64-linux-gnu/bitcoin-253a691441dd-aarch64-linux-gnu-debug.tar.gz
4a6ec8df80822e5938b1a55f2e08a945b12a9d4f8b2f1bad05ea8be90a3b6173  guix-build-253a691441dd/output/aarch64-linux-gnu/bitcoin-253a691441dd-aarch64-linux-gnu.tar.gz
48de6fea5e2d35be40534dedfc51e30e759366b5b6cb6b14e053dbb5cce1c183  guix-build-253a691441dd/output/arm-linux-gnueabihf/SHA256SUMS.part
698a07e7d4cc5b98e64fec240735c7982ccd64abd30ef39eb09f3e1bf10627e7  guix-build-253a691441dd/output/arm-linux-gnueabihf/bitcoin-253a691441dd-arm-linux-gnueabihf-debug.tar.gz
1d33f5c6dab8dea66801ef49c33217bd538fd76d7a1a97cd6609530b94f71805  guix-build-253a691441dd/output/arm-linux-gnueabihf/bitcoin-253a691441dd-arm-linux-gnueabihf.tar.gz
91e2daf2c6fd4c1a97b51e33ee8699a747c16dad2ad27dc3a1e5de0ca6385dbc  guix-build-253a691441dd/output/arm64-apple-darwin/SHA256SUMS.part
b2c439dec7f332d8d46b6473e3015095084297cc871f91eb368889c999a930b5  guix-build-253a691441dd/output/arm64-apple-darwin/bitcoin-253a691441dd-arm64-apple-darwin-unsigned.tar.gz
aa85020faab673502192ccf3ab0dfb972df786961317d608c66092729baed133  guix-build-253a691441dd/output/arm64-apple-darwin/bitcoin-253a691441dd-arm64-apple-darwin-unsigned.zip
3a3fce1fd98d6670006daf414ac6bb5ea73336f7846d076606b1c0f0601b09a9  guix-build-253a691441dd/output/arm64-apple-darwin/bitcoin-253a691441dd-arm64-apple-darwin.tar.gz
943a992518ab2b54e93316e89c5ac65e7250004756ac9b37277af5d267e157a8  guix-build-253a691441dd/output/dist-archive/bitcoin-253a691441dd.tar.gz
94088a64cc9d06bea8dd591cbf140f9e4ef7727532254c5b5e0614d5178780f4  guix-build-253a691441dd/output/powerpc64-linux-gnu/SHA256SUMS.part
a7f2db2812ec521162c9e06a8206d76f65ec8fd6be6c11a2d6bf53b863e93577  guix-build-253a691441dd/output/powerpc64-linux-gnu/bitcoin-253a691441dd-powerpc64-linux-gnu-debug.tar.gz
9081f9d3bbdea50045253fddd390aa657c505c74eb6654458cba245a6ef13870  guix-build-253a691441dd/output/powerpc64-linux-gnu/bitcoin-253a691441dd-powerpc64-linux-gnu.tar.gz
efd4c7418f08418adb975790f52222c700ab59851555a3e8e0f59579d2aebebe  guix-build-253a691441dd/output/riscv64-linux-gnu/SHA256SUMS.part
e385f0716031f7e6681a93add935cf29345d15909ee3993b7725cc89b260dcb4  guix-build-253a691441dd/output/riscv64-linux-gnu/bitcoin-253a691441dd-riscv64-linux-gnu-debug.tar.gz
4c6aec6956291cc28ec5d46627cfb02456ab223ba7a3b4528956b41f4868509d  guix-build-253a691441dd/output/riscv64-linux-gnu/bitcoin-253a691441dd-riscv64-linux-gnu.tar.gz
aa4e835f3356675741c76634d71142bbf08cf082476c23bbb2b32fb039ec8181  guix-build-253a691441dd/output/x86_64-apple-darwin/SHA256SUMS.part
12c1a3964f2ba89ba9f19c8e3ec9b70eacbb7d2e76d3cfc13a12afc4f1dd29c9  guix-build-253a691441dd/output/x86_64-apple-darwin/bitcoin-253a691441dd-x86_64-apple-darwin-unsigned.tar.gz
0832fdf8af0c9512d2ed0aa51cfe967c48bafb39f98ab370f53837ffbdf908cc  guix-build-253a691441dd/output/x86_64-apple-darwin/bitcoin-253a691441dd-x86_64-apple-darwin-unsigned.zip
8e80b5391bd2271a7791e43ca3869e964d35fa57ad4acdf07e81774c0bc30836  guix-build-253a691441dd/output/x86_64-apple-darwin/bitcoin-253a691441dd-x86_64-apple-darwin.tar.gz
e5c9a4e491818cd9b87f94c061eb381101a603c1136ac100fda940e000db94a6  guix-build-253a691441dd/output/x86_64-linux-gnu/SHA256SUMS.part
75b6c5dcf8993e264e2c5269521bf1334573ee3d6cf02aaabd9e6d062f9aecf9  guix-build-253a691441dd/output/x86_64-linux-gnu/bitcoin-253a691441dd-x86_64-linux-gnu-debug.tar.gz
a7169d35d96420260c9e2c2156c66f2130cfbdf649febad01627602306c55b74  guix-build-253a691441dd/output/x86_64-linux-gnu/bitcoin-253a691441dd-x86_64-linux-gnu.tar.gz
eefafa6cfbc56e44530634978ea114626693fe8a6b31e14f192b7e45173ff1fe  guix-build-253a691441dd/output/x86_64-w64-mingw32/SHA256SUMS.part
279eb3acd6ccb3aa1362d07a37dd2fdeb877fac5d673620a405fa887806d8e2c  guix-build-253a691441dd/output/x86_64-w64-mingw32/bitcoin-253a691441dd-win64-debug.zip
1524efbd8e36b3b06ecfbdd5e0889b5c06c25c82ef782671725fc8068dd7eb1a  guix-build-253a691441dd/output/x86_64-w64-mingw32/bitcoin-253a691441dd-win64-setup-unsigned.exe
037a06e7ef58c70681b6415584fd61f67cc266bc25db89a5935f1a937bbf86db  guix-build-253a691441dd/output/x86_64-w64-mingw32/bitcoin-253a691441dd-win64-unsigned.tar.gz
05f61faaf023e70a420a6c6bf9fd5b0fb7e123b725e7df076e6593f5d9a6fa33  guix-build-253a691441dd/output/x86_64-w64-mingw32/bitcoin-253a691441dd-win64.zip
```

hebasto commented:
My Guix build:
```
aarch64
d4baa1d2e9ee7abeb60d115dd52c6bc7d02e8116fe4b94c0f040371321e37b80  guix-build-253a691441dd/output/aarch64-linux-gnu/SHA256SUMS.part
7c38410b9468799b74371ba195f34121a4789e917dcf611f2ccc4b81f8a63eee  guix-build-253a691441dd/output/aarch64-linux-gnu/bitcoin-253a691441dd-aarch64-linux-gnu-debug.tar.gz
4a6ec8df80822e5938b1a55f2e08a945b12a9d4f8b2f1bad05ea8be90a3b6173  guix-build-253a691441dd/output/aarch64-linux-gnu/bitcoin-253a691441dd-aarch64-linux-gnu.tar.gz
48de6fea5e2d35be40534dedfc51e30e759366b5b6cb6b14e053dbb5cce1c183  guix-build-253a691441dd/output/arm-linux-gnueabihf/SHA256SUMS.part
698a07e7d4cc5b98e64fec240735c7982ccd64abd30ef39eb09f3e1bf10627e7  guix-build-253a691441dd/output/arm-linux-gnueabihf/bitcoin-253a691441dd-arm-linux-gnueabihf-debug.tar.gz
1d33f5c6dab8dea66801ef49c33217bd538fd76d7a1a97cd6609530b94f71805  guix-build-253a691441dd/output/arm-linux-gnueabihf/bitcoin-253a691441dd-arm-linux-gnueabihf.tar.gz
91e2daf2c6fd4c1a97b51e33ee8699a747c16dad2ad27dc3a1e5de0ca6385dbc  guix-build-253a691441dd/output/arm64-apple-darwin/SHA256SUMS.part
b2c439dec7f332d8d46b6473e3015095084297cc871f91eb368889c999a930b5  guix-build-253a691441dd/output/arm64-apple-darwin/bitcoin-253a691441dd-arm64-apple-darwin-unsigned.tar.gz
aa85020faab673502192ccf3ab0dfb972df786961317d608c66092729baed133  guix-build-253a691441dd/output/arm64-apple-darwin/bitcoin-253a691441dd-arm64-apple-darwin-unsigned.zip
3a3fce1fd98d6670006daf414ac6bb5ea73336f7846d076606b1c0f0601b09a9  guix-build-253a691441dd/output/arm64-apple-darwin/bitcoin-253a691441dd-arm64-apple-darwin.tar.gz
943a992518ab2b54e93316e89c5ac65e7250004756ac9b37277af5d267e157a8  guix-build-253a691441dd/output/dist-archive/bitcoin-253a691441dd.tar.gz
94088a64cc9d06bea8dd591cbf140f9e4ef7727532254c5b5e0614d5178780f4  guix-build-253a691441dd/output/powerpc64-linux-gnu/SHA256SUMS.part
a7f2db2812ec521162c9e06a8206d76f65ec8fd6be6c11a2d6bf53b863e93577  guix-build-253a691441dd/output/powerpc64-linux-gnu/bitcoin-253a691441dd-powerpc64-linux-gnu-debug.tar.gz
9081f9d3bbdea50045253fddd390aa657c505c74eb6654458cba245a6ef13870  guix-build-253a691441dd/output/powerpc64-linux-gnu/bitcoin-253a691441dd-powerpc64-linux-gnu.tar.gz
efd4c7418f08418adb975790f52222c700ab59851555a3e8e0f59579d2aebebe  guix-build-253a691441dd/output/riscv64-linux-gnu/SHA256SUMS.part
e385f0716031f7e6681a93add935cf29345d15909ee3993b7725cc89b260dcb4  guix-build-253a691441dd/output/riscv64-linux-gnu/bitcoin-253a691441dd-riscv64-linux-gnu-debug.tar.gz
4c6aec6956291cc28ec5d46627cfb02456ab223ba7a3b4528956b41f4868509d  guix-build-253a691441dd/output/riscv64-linux-gnu/bitcoin-253a691441dd-riscv64-linux-gnu.tar.gz
aa4e835f3356675741c76634d71142bbf08cf082476c23bbb2b32fb039ec8181  guix-build-253a691441dd/output/x86_64-apple-darwin/SHA256SUMS.part
12c1a3964f2ba89ba9f19c8e3ec9b70eacbb7d2e76d3cfc13a12afc4f1dd29c9  guix-build-253a691441dd/output/x86_64-apple-darwin/bitcoin-253a691441dd-x86_64-apple-darwin-unsigned.tar.gz
0832fdf8af0c9512d2ed0aa51cfe967c48bafb39f98ab370f53837ffbdf908cc  guix-build-253a691441dd/output/x86_64-apple-darwin/bitcoin-253a691441dd-x86_64-apple-darwin-unsigned.zip
8e80b5391bd2271a7791e43ca3869e964d35fa57ad4acdf07e81774c0bc30836  guix-build-253a691441dd/output/x86_64-apple-darwin/bitcoin-253a691441dd-x86_64-apple-darwin.tar.gz
e5c9a4e491818cd9b87f94c061eb381101a603c1136ac100fda940e000db94a6  guix-build-253a691441dd/output/x86_64-linux-gnu/SHA256SUMS.part
75b6c5dcf8993e264e2c5269521bf1334573ee3d6cf02aaabd9e6d062f9aecf9  guix-build-253a691441dd/output/x86_64-linux-gnu/bitcoin-253a691441dd-x86_64-linux-gnu-debug.tar.gz
a7169d35d96420260c9e2c2156c66f2130cfbdf649febad01627602306c55b74  guix-build-253a691441dd/output/x86_64-linux-gnu/bitcoin-253a691441dd-x86_64-linux-gnu.tar.gz
eefafa6cfbc56e44530634978ea114626693fe8a6b31e14f192b7e45173ff1fe  guix-build-253a691441dd/output/x86_64-w64-mingw32/SHA256SUMS.part
279eb3acd6ccb3aa1362d07a37dd2fdeb877fac5d673620a405fa887806d8e2c  guix-build-253a691441dd/output/x86_64-w64-mingw32/bitcoin-253a691441dd-win64-debug.zip
1524efbd8e36b3b06ecfbdd5e0889b5c06c25c82ef782671725fc8068dd7eb1a  guix-build-253a691441dd/output/x86_64-w64-mingw32/bitcoin-253a691441dd-win64-setup-unsigned.exe
037a06e7ef58c70681b6415584fd61f67cc266bc25db89a5935f1a937bbf86db  guix-build-253a691441dd/output/x86_64-w64-mingw32/bitcoin-253a691441dd-win64-unsigned.tar.gz
05f61faaf023e70a420a6c6bf9fd5b0fb7e123b725e7df076e6593f5d9a6fa33  guix-build-253a691441dd/output/x86_64-w64-mingw32/bitcoin-253a691441dd-win64.zip
```

hebasto reviewed (approved):
ACK 253a691441ddb04eac2b318112064e69f4b837b7.

hebasto commented:
@theStack 

Could rebase once more please to refresh the CI?

hebasto reviewed (approved):
re-ACK 807a429a33cd9b1986f1ebb8fd761887f556afd8, rebased only since my recent [review](https://github.com/bitcoin/bitcoin/pull/30856#pullrequestreview-2300357639).

My Guix build:
```
aarch64
f6a91156bc9ac2ec9367d75a433d903db4fe5c4ff0c5680289882f1ca58b9759  guix-build-807a429a33cd/output/aarch64-linux-gnu/SHA256SUMS.part
2704a2fbc76888fb9073631513ea1fc2203edb344814dbe5db02fd51c6d9949d  guix-build-807a429a33cd/output/aarch64-linux-gnu/bitcoin-807a429a33cd-aarch64-linux-gnu-debug.tar.gz
a5cd59cb03c5df4eb20b614ead7f64e1e3843924b70f5f33023e78d0a3625f1a  guix-build-807a429a33cd/output/aarch64-linux-gnu/bitcoin-807a429a33cd-aarch64-linux-gnu.tar.gz
d6b3372d5c9105cd27b31fb60fca5290c988f1ba6ff05f25f4b0f93b6202e451  guix-build-807a429a33cd/output/arm-linux-gnueabihf/SHA256SUMS.part
b466d2d031ab5d96d984a9be9839cc50d4c146b71d0f551e088d5132b73ee218  guix-build-807a429a33cd/output/arm-linux-gnueabihf/bitcoin-807a429a33cd-arm-linux-gnueabihf-debug.tar.gz
43b47b6cb243aab6c54c7d6234ab7db83b381a3006874294cce106494f1ed8a9  guix-build-807a429a33cd/output/arm-linux-gnueabihf/bitcoin-807a429a33cd-arm-linux-gnueabihf.tar.gz
c05ccde98443c4ec9e6e792fd5fdfac63f46a8d518eb6937516c988d124995f3  guix-build-807a429a33cd/output/arm64-apple-darwin/SHA256SUMS.part
1170f7ceec52a6a6b942e98d7a21e001a0941c2ce57b06068420b09a957f2602  guix-build-807a429a33cd/output/arm64-apple-darwin/bitcoin-807a429a33cd-arm64-apple-darwin-unsigned.tar.gz
e15922f916e5a9937b81eb9d103462650dcc46bada100f4f1a80511166207e56  guix-build-807a429a33cd/output/arm64-apple-darwin/bitcoin-807a429a33cd-arm64-apple-darwin-unsigned.zip
990b9725acf3fa985dd351932d748095d2cc56a8d6d460042ce68ccc2bce913e  guix-build-807a429a33cd/output/arm64-apple-darwin/bitcoin-807a429a33cd-arm64-apple-darwin.tar.gz
ace1924445a06db05bc63a79fa162bd02c654069308098e3169273fe84c0401d  guix-build-807a429a33cd/output/dist-archive/bitcoin-807a429a33cd.tar.gz
6ecb2cf38ac08c865c30521335b3d9816d9b1dbeaf5c2e2301a9dbc0efb8b83d  guix-build-807a429a33cd/output/powerpc64-linux-gnu/SHA256SUMS.part
f6d82f9ba0965dbcf4b69b678b0b16647bb4dfcb44b8f325629cfbcc11a90a9f  guix-build-807a429a33cd/output/powerpc64-linux-gnu/bitcoin-807a429a33cd-powerpc64-linux-gnu-debug.tar.gz
4c6ea3b643afa85225881fc038e513e06b0a277f374d2af98a3f5d5413eabdeb  guix-build-807a429a33cd/output/powerpc64-linux-gnu/bitcoin-807a429a33cd-powerpc64-linux-gnu.tar.gz
7887479bb46ba88ea44f392117e6a821ce38756e55d128c89bfd1dcb696c64a6  guix-build-807a429a33cd/output/riscv64-linux-gnu/SHA256SUMS.part
8d9a8b0810efe2ad59f5047c3fcf7625d1d3d0dfb9728c417ccf614184baf28d  guix-build-807a429a33cd/output/riscv64-linux-gnu/bitcoin-807a429a33cd-riscv64-linux-gnu-debug.tar.gz
ccd28d502db7f25c74a06ab86bb24eb85351d84909b9d6baa025f89c4e907438  guix-build-807a429a33cd/output/riscv64-linux-gnu/bitcoin-807a429a33cd-riscv64-linux-gnu.tar.gz
a060ef9345d88e11b7b19c57984146e4e9ff27949359ed3df509fe8ec74744c3  guix-build-807a429a33cd/output/x86_64-apple-darwin/SHA256SUMS.part
56983cf5081c7afee7d748cf4b11c8ae644d2d53607d32fd9049300c54521217  guix-build-807a429a33cd/output/x86_64-apple-darwin/bitcoin-807a429a33cd-x86_64-apple-darwin-unsigned.tar.gz
28df06a998b94ece5832d3741a218184379e4ad0f244d51d25e2cbd3b4c11276  guix-build-807a429a33cd/output/x86_64-apple-darwin/bitcoin-807a429a33cd-x86_64-apple-darwin-unsigned.zip
5d339c04c55f5f532249c39e50c06a37d1f73cafa2db56ca180373432d002b66  guix-build-807a429a33cd/output/x86_64-apple-darwin/bitcoin-807a429a33cd-x86_64-apple-darwin.tar.gz
7d0e41f4863d3f7a18e50722c722f516653a4dca7963c6440c3b071a8bc5e3fc  guix-build-807a429a33cd/output/x86_64-linux-gnu/SHA256SUMS.part
cf991810945fc14610bb309386d61c439bd59c62ba00e39bcc44482070c55415  guix-build-807a429a33cd/output/x86_64-linux-gnu/bitcoin-807a429a33cd-x86_64-linux-gnu-debug.tar.gz
5f13845e9cc99e8dc917e325ae100d5ff66f3d63b8beed53d2e89051a87b6d17  guix-build-807a429a33cd/output/x86_64-linux-gnu/bitcoin-807a429a33cd-x86_64-linux-gnu.tar.gz
cbf166153938607b190c01ade9eb831d1241144c5064251ea7d7df7542779776  guix-build-807a429a33cd/output/x86_64-w64-mingw32/SHA256SUMS.part
99420aefea0c34b7d61745b54f256cb2ad8ab29c9cbbe22b2b543b0829c67ef4  guix-build-807a429a33cd/output/x86_64-w64-mingw32/bitcoin-807a429a33cd-win64-debug.zip
245f5b169385d586fa87e2734a94234296f2c8d5232fb6e9c29e5c41f1bfdc89  guix-build-807a429a33cd/output/x86_64-w64-mingw32/bitcoin-807a429a33cd-win64-setup-unsigned.exe
ebea2cca9f194b7f727859c8f9a56649c204239f43dbd49096d43af8a38cbb42  guix-build-807a429a33cd/output/x86_64-w64-mingw32/bitcoin-807a429a33cd-win64-unsigned.tar.gz
6ad3b82009309c5063e666f2e57c35e4bef14809752117ce092255055c660ce0  guix-build-807a429a33cd/output/x86_64-w64-mingw32/bitcoin-807a429a33cd-win64.zip
```

hebasto commented on src/CMakeLists.txt:
```diff
@@ -8,23 +8,15 @@ include(AddWindowsResources)
 configure_file(${PROJECT_SOURCE_DIR}/cmake/bitcoin-config.h.in config/bitcoin-config.h USE_SOURCE_PERMISSIONS @ONLY)
 include_directories(${CMAKE_CURRENT_BINARY_DIR} ${CMAKE_CURRENT_SOURCE_DIR})
 
-# TODO: After the transition from Autotools to CMake, the obj/ subdirectory
-#       could be dropped as its only purpose was to separate a generated header
-#       from source files.
```
@theuni 

You might be interesting in reviewing this PR, as this comment was added at your request while working on the CMake staging branch, if I recall correctly :)

theuni reviewed (commented):
Concept ACK.

Though, `obj/` was acting as a bit of a namespace here (IIRC, that was another reason for it to be in a subdir). Especially since this is included with `<>` as opposed to `""`, while we're at it, I think we should rename this file. `build.h` seems generic enough to collide with something else. `bitcoin-build.h` maybe?

theuni commented:
Likewise for `config/bitcoin-config.h` btw. IIRC `config/` part is an old artifact of the switch _to_ autotools. I don't see any reason why bitcoin-config.h shouldn't just live in `build/src/` now.

Err.. I guess that would mean touching a _lot_ of files though :)

maflcko reviewed (approved):
lgtm

Maybe squash, to avoid changing the file name twice?

theuni reviewed (approved):
utACK 7025942687fd5e91d0a10ce5b2ac673b67a63491

fanquake commented:
Guix build (x86_64):
```bash
f65f7839cb8ce4b194ab55a698b9840dd335d790a5499d3303cf79885405bedb  guix-build-7025942687fd/output/aarch64-linux-gnu/SHA256SUMS.part
7953c080586b3500c73edd93731b8b9bcfd79ac4f5fe7953d51dc54643899255  guix-build-7025942687fd/output/aarch64-linux-gnu/bitcoin-7025942687fd-aarch64-linux-gnu-debug.tar.gz
c82339ac1813727df996deda765087d430080a62d5fff7e16f46727be703a68c  guix-build-7025942687fd/output/aarch64-linux-gnu/bitcoin-7025942687fd-aarch64-linux-gnu.tar.gz
ffe2b02e6187d2c04e3d9c22359448cdacab537794ebd19d5f3b0ccd907a915a  guix-build-7025942687fd/output/arm-linux-gnueabihf/SHA256SUMS.part
a5765dcee8bf4f0901ecc3821408f2f9429720c1fa62566c2789e332c151681b  guix-build-7025942687fd/output/arm-linux-gnueabihf/bitcoin-7025942687fd-arm-linux-gnueabihf-debug.tar.gz
cdc734e3c0f1801688bdedb006da10a2e6632fadbdd59e604df274d07e887c27  guix-build-7025942687fd/output/arm-linux-gnueabihf/bitcoin-7025942687fd-arm-linux-gnueabihf.tar.gz
ba20403a2d49bde31172d25145a3f1f8d0fb6eb45721412ab591780e2fede9b4  guix-build-7025942687fd/output/arm64-apple-darwin/SHA256SUMS.part
4bb2e1643b9f496ed6a46af49c4c9246c3efe91b48ac7da290bd902abd893e8d  guix-build-7025942687fd/output/arm64-apple-darwin/bitcoin-7025942687fd-arm64-apple-darwin-unsigned.tar.gz
8670598c71bfbe86770231d11dece2d5d43544f120be4287c0ff5017712b09b3  guix-build-7025942687fd/output/arm64-apple-darwin/bitcoin-7025942687fd-arm64-apple-darwin-unsigned.zip
40c783b17d607bae4e5e0f6e71a0dfb1e55e4ff4951ebdce981bdd8e72edad4b  guix-build-7025942687fd/output/arm64-apple-darwin/bitcoin-7025942687fd-arm64-apple-darwin.tar.gz
e45543e43185d2b35a69ff6177340077e34064b922ac1f6f50bc152f4eaed806  guix-build-7025942687fd/output/dist-archive/bitcoin-7025942687fd.tar.gz
02793c2b05814c10fd900e659d753a383333c89e0c8a4d57daec556454fff3a3  guix-build-7025942687fd/output/powerpc64-linux-gnu/SHA256SUMS.part
7d4434d13c53c27d1e2f34767f373a026181e4a72f4613d3bdba8d3f4812961a  guix-build-7025942687fd/output/powerpc64-linux-gnu/bitcoin-7025942687fd-powerpc64-linux-gnu-debug.tar.gz
14bc762c2429377069b49146fde9712747faf4fcfcad06a94e8c27abd32c0b4e  guix-build-7025942687fd/output/powerpc64-linux-gnu/bitcoin-7025942687fd-powerpc64-linux-gnu.tar.gz
d8534437521bdfbdd01c36ccf8f5da28fc9677098aaadbeaaffcc5aec992150e  guix-build-7025942687fd/output/riscv64-linux-gnu/SHA256SUMS.part
243ad7cabe6cf8778970a83a977a3fce81c9086385e06097bfbefc2d32fab92e  guix-build-7025942687fd/output/riscv64-linux-gnu/bitcoin-7025942687fd-riscv64-linux-gnu-debug.tar.gz
0d01e1d1cd9b3bb03ed9c4b08e2dae8d46eeac29c1105d13825e95917f144883  guix-build-7025942687fd/output/riscv64-linux-gnu/bitcoin-7025942687fd-riscv64-linux-gnu.tar.gz
eef35730f0a47bb3697ead2fe513e9f765c37fc06f4b2f2c053865ec6965c6a9  guix-build-7025942687fd/output/x86_64-apple-darwin/SHA256SUMS.part
d88c34c6a87d008a45ebb8c48c0c42c96a8e228d72826fbdef80f4a2676fd67f  guix-build-7025942687fd/output/x86_64-apple-darwin/bitcoin-7025942687fd-x86_64-apple-darwin-unsigned.tar.gz
d6298d8dd0a25e343b09d4cce735c7493aba7bf80ad95b863dafdfce53235749  guix-build-7025942687fd/output/x86_64-apple-darwin/bitcoin-7025942687fd-x86_64-apple-darwin-unsigned.zip
803923f3e20ac251b15c5c9ccf18e6eeb8c7deb828b45bee7c9dcaf15363222f  guix-build-7025942687fd/output/x86_64-apple-darwin/bitcoin-7025942687fd-x86_64-apple-darwin.tar.gz
b50f1e5000d5301d481b9d0bceb74aa358500444489c87d9e8a81e0ba8886e79  guix-build-7025942687fd/output/x86_64-linux-gnu/SHA256SUMS.part
d95587edae05630ad2a126b25c3b1fe41b5fe2cc0b8de5474e0c6815f498f8ba  guix-build-7025942687fd/output/x86_64-linux-gnu/bitcoin-7025942687fd-x86_64-linux-gnu-debug.tar.gz
bad3de3cff3467c6de928cda6dd554a5f4b974f7ae449713c0f5199450af909e  guix-build-7025942687fd/output/x86_64-linux-gnu/bitcoin-7025942687fd-x86_64-linux-gnu.tar.gz
dfcf9ad8bf6615d00ee393cd1ae9b2123c8e86701d75a3e9bfdb73b094693c93  guix-build-7025942687fd/output/x86_64-w64-mingw32/SHA256SUMS.part
969ff114447e139dcf4f6c3c4d20ac8b1f1e41affc5f6797bae1de8f59da0e63  guix-build-7025942687fd/output/x86_64-w64-mingw32/bitcoin-7025942687fd-win64-debug.zip
523c24b241dc6e55603266344bd0aad47aed526543a22ed37760aba29bb5759b  guix-build-7025942687fd/output/x86_64-w64-mingw32/bitcoin-7025942687fd-win64-setup-unsigned.exe
8ce71925b01017e2aa85e36559635a739bc5306d9d28ff81a77d1f79b8a6ba4d  guix-build-7025942687fd/output/x86_64-w64-mingw32/bitcoin-7025942687fd-win64-unsigned.tar.gz
61739b2e4af4dcf58af182088325c45ed1aa9103dcec8dfe29252d25f2203049  guix-build-7025942687fd/output/x86_64-w64-mingw32/bitcoin-7025942687fd-win64.zip
```

hebasto reviewed (commented):
My Guix build:
```
aarch64
f65f7839cb8ce4b194ab55a698b9840dd335d790a5499d3303cf79885405bedb  guix-build-7025942687fd/output/aarch64-linux-gnu/SHA256SUMS.part
7953c080586b3500c73edd93731b8b9bcfd79ac4f5fe7953d51dc54643899255  guix-build-7025942687fd/output/aarch64-linux-gnu/bitcoin-7025942687fd-aarch64-linux-gnu-debug.tar.gz
c82339ac1813727df996deda765087d430080a62d5fff7e16f46727be703a68c  guix-build-7025942687fd/output/aarch64-linux-gnu/bitcoin-7025942687fd-aarch64-linux-gnu.tar.gz
ffe2b02e6187d2c04e3d9c22359448cdacab537794ebd19d5f3b0ccd907a915a  guix-build-7025942687fd/output/arm-linux-gnueabihf/SHA256SUMS.part
a5765dcee8bf4f0901ecc3821408f2f9429720c1fa62566c2789e332c151681b  guix-build-7025942687fd/output/arm-linux-gnueabihf/bitcoin-7025942687fd-arm-linux-gnueabihf-debug.tar.gz
cdc734e3c0f1801688bdedb006da10a2e6632fadbdd59e604df274d07e887c27  guix-build-7025942687fd/output/arm-linux-gnueabihf/bitcoin-7025942687fd-arm-linux-gnueabihf.tar.gz
ba20403a2d49bde31172d25145a3f1f8d0fb6eb45721412ab591780e2fede9b4  guix-build-7025942687fd/output/arm64-apple-darwin/SHA256SUMS.part
4bb2e1643b9f496ed6a46af49c4c9246c3efe91b48ac7da290bd902abd893e8d  guix-build-7025942687fd/output/arm64-apple-darwin/bitcoin-7025942687fd-arm64-apple-darwin-unsigned.tar.gz
8670598c71bfbe86770231d11dece2d5d43544f120be4287c0ff5017712b09b3  guix-build-7025942687fd/output/arm64-apple-darwin/bitcoin-7025942687fd-arm64-apple-darwin-unsigned.zip
40c783b17d607bae4e5e0f6e71a0dfb1e55e4ff4951ebdce981bdd8e72edad4b  guix-build-7025942687fd/output/arm64-apple-darwin/bitcoin-7025942687fd-arm64-apple-darwin.tar.gz
e45543e43185d2b35a69ff6177340077e34064b922ac1f6f50bc152f4eaed806  guix-build-7025942687fd/output/dist-archive/bitcoin-7025942687fd.tar.gz
02793c2b05814c10fd900e659d753a383333c89e0c8a4d57daec556454fff3a3  guix-build-7025942687fd/output/powerpc64-linux-gnu/SHA256SUMS.part
7d4434d13c53c27d1e2f34767f373a026181e4a72f4613d3bdba8d3f4812961a  guix-build-7025942687fd/output/powerpc64-linux-gnu/bitcoin-7025942687fd-powerpc64-linux-gnu-debug.tar.gz
14bc762c2429377069b49146fde9712747faf4fcfcad06a94e8c27abd32c0b4e  guix-build-7025942687fd/output/powerpc64-linux-gnu/bitcoin-7025942687fd-powerpc64-linux-gnu.tar.gz
d8534437521bdfbdd01c36ccf8f5da28fc9677098aaadbeaaffcc5aec992150e  guix-build-7025942687fd/output/riscv64-linux-gnu/SHA256SUMS.part
243ad7cabe6cf8778970a83a977a3fce81c9086385e06097bfbefc2d32fab92e  guix-build-7025942687fd/output/riscv64-linux-gnu/bitcoin-7025942687fd-riscv64-linux-gnu-debug.tar.gz
0d01e1d1cd9b3bb03ed9c4b08e2dae8d46eeac29c1105d13825e95917f144883  guix-build-7025942687fd/output/riscv64-linux-gnu/bitcoin-7025942687fd-riscv64-linux-gnu.tar.gz
eef35730f0a47bb3697ead2fe513e9f765c37fc06f4b2f2c053865ec6965c6a9  guix-build-7025942687fd/output/x86_64-apple-darwin/SHA256SUMS.part
d88c34c6a87d008a45ebb8c48c0c42c96a8e228d72826fbdef80f4a2676fd67f  guix-build-7025942687fd/output/x86_64-apple-darwin/bitcoin-7025942687fd-x86_64-apple-darwin-unsigned.tar.gz
d6298d8dd0a25e343b09d4cce735c7493aba7bf80ad95b863dafdfce53235749  guix-build-7025942687fd/output/x86_64-apple-darwin/bitcoin-7025942687fd-x86_64-apple-darwin-unsigned.zip
803923f3e20ac251b15c5c9ccf18e6eeb8c7deb828b45bee7c9dcaf15363222f  guix-build-7025942687fd/output/x86_64-apple-darwin/bitcoin-7025942687fd-x86_64-apple-darwin.tar.gz
b50f1e5000d5301d481b9d0bceb74aa358500444489c87d9e8a81e0ba8886e79  guix-build-7025942687fd/output/x86_64-linux-gnu/SHA256SUMS.part
d95587edae05630ad2a126b25c3b1fe41b5fe2cc0b8de5474e0c6815f498f8ba  guix-build-7025942687fd/output/x86_64-linux-gnu/bitcoin-7025942687fd-x86_64-linux-gnu-debug.tar.gz
bad3de3cff3467c6de928cda6dd554a5f4b974f7ae449713c0f5199450af909e  guix-build-7025942687fd/output/x86_64-linux-gnu/bitcoin-7025942687fd-x86_64-linux-gnu.tar.gz
dfcf9ad8bf6615d00ee393cd1ae9b2123c8e86701d75a3e9bfdb73b094693c93  guix-build-7025942687fd/output/x86_64-w64-mingw32/SHA256SUMS.part
969ff114447e139dcf4f6c3c4d20ac8b1f1e41affc5f6797bae1de8f59da0e63  guix-build-7025942687fd/output/x86_64-w64-mingw32/bitcoin-7025942687fd-win64-debug.zip
523c24b241dc6e55603266344bd0aad47aed526543a22ed37760aba29bb5759b  guix-build-7025942687fd/output/x86_64-w64-mingw32/bitcoin-7025942687fd-win64-setup-unsigned.exe
8ce71925b01017e2aa85e36559635a739bc5306d9d28ff81a77d1f79b8a6ba4d  guix-build-7025942687fd/output/x86_64-w64-mingw32/bitcoin-7025942687fd-win64-unsigned.tar.gz
61739b2e4af4dcf58af182088325c45ed1aa9103dcec8dfe29252d25f2203049  guix-build-7025942687fd/output/x86_64-w64-mingw32/bitcoin-7025942687fd-win64.zip
```

hebasto reviewed (commented):
Post-merge re-ACK 7025942687fd5e91d0a10ce5b2ac673b67a63491.
